addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -791,6 +792,7 @@ export class Orderline extends PosModel {
             price_type: this.price_type,
             combo_line_ids: this.comboLines?.map((line) => line.id || line.cid),
             combo_parent_id: this.comboParent?.id || this.comboParent?.cid,
+            comboLine_id: this.comboLine?.id,
```

Comment:
```suggestion
            combo_line_id: this.comboLine?.id,
```

rename to combo_line_id so that it's already compatible for the upcoming fix of saving it in the database.